tests/resources/dspa-lite.yaml

```diff
@@ -60,7 +60,7 @@ spec:
   database:
     mariaDB:
       deploy: true
-      image: quay.io/centos7/mariadb-103-centos7:ea07c0dade9571d78a272b453fd2dea92077dc7f
+      image: quay.io/sclorg/mariadb-105-c9s:latest
```

Comment:
hrmm I also see: https://quay.io/repository/mariadb-foundation/mariadb-devel?tab=tags which looks like a more official source, do you know what the difference is?

Reply:
@HumairAK not sure but the SCL Org is a Red Hat project:
https://github.com/sclorg

Reply:
works for me